src/main/java/web/cinema/security/impl/CustomeUserDetailService.java

```diff
@@ -0,0 +1,34 @@
+package web.cinema.security.impl;
+
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+import web.cinema.model.User;
+import web.cinema.service.UserService;
+
+@Service
+public class CustomeUserDetailService implements UserDetailsService {
+    private final UserService userService;
+
+    public CustomeUserDetailService(UserService userService) {
+        this.userService = userService;
+    }
+
+    @Override
+    public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
+        User user = userService.findByEmail(email);
+        org.springframework.security.core.userdetails.User.UserBuilder userBuilder;
```

Comment:
```suggestion
        UserBuilder userBuilder;
```
давай зайве винесемо в імпорт